extensions/wikia/WikiaMaps/controllers/WikiaMapsSpecialController.class.php

```diff
@@ -319,8 +319,16 @@ public function translation() {
 	 * @param Integer $height
 	 */
 	private function convertImagesToThumbs( &$items, $width, $height ) {
+		/*
+		 FIXME this is a temporary fix for MWEB-1100
+		 We have 404s cached for a week and need to add cache buster until they are expired
+		 This should be removed on 2015-02-16 (first Monday after expiration time)
+		 */
+		$cacheBuster = '?cb20150206';
```

Comment:
How about `?cb=20150206`? It probably doesn't make a difference but looks better.